### DISCUSSION TITLE
有没有关于链式调用的详细文档呢？

### DISCUSSION THREAD

**Discussion by krvipsoft** [slowlyo/owl-admin#120] [FAQ]
最近初步接触这个框架，感觉很方便，节省很多开发时间。但是缺少增删改查链式调用的细节参数，希望能完善下文档。
比如：amis()->TableColumn  里面都能链接哪些参数，比如 amis()->TableColumn->type() type都有哪些类型，都是在什么情况下使用的

**Comment by slowlyo** [comment]
- [组件使用](https://doc.owladmin.com/#/guide/component-use)
- [框架原理](https://doc.owladmin.com/#/theory/frame)
- [组件文档](https://aisuda.bce.baidu.com/amis/zh-CN/components/page)

把前两个文档看明白了, 然后根据百度官方的组件文档用就行了

由于是个人在维护, 没有太多精力去细化文档😅
